pkg/db/migration/000011_init.down.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TYPE CONNECTOR_VALID_CONNECTOR_TYPE RENAME VALUE 'CONNECTOR_TYPE_APPLICATION' TO 'CONNECTOR_TYPE_BLOCKCHAIN';
```

Comment:
@donch1989 I didn't see any `down` migrations but still I wrote this one. Is there an issue with migrating down?

Reply:
For us, the migration down is not essential for now, but I recall that the library will raise an error if we don't provide the down file. Therefore, we simply create empty ones.